pep-0008.txt

```diff
@@ -241,6 +241,17 @@ Another such case is with ``assert`` statements.
 
 Make sure to indent the continued line appropriately.
 
+
+Exception Message String Literals
+---------------------------------
```

Comment:
Maybe there's a better place to put this into PEP 8, but this was my first thought after skimming through.

Reply:
Seems like the best spot to me :+1: